packages/fxa-auth-server/lib/subhub/updates.js

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. 
+*/
+
+'use strict';
+
+module.exports = function (log) {
+
+  return function start(messageQueue, push, db) {
+
+    function handleProfileUpdated(message) {
+      const uid = message && message.uid;
```

Comment:
A couple of questions here: 

* Should we do validation on these messages to ensure they are well formed?
* Will there ever be multiple message types?

Reply:
Yes, sorry, still copypasta.